src/relay-runtime/src/RelayRuntimeTypes.js

```diff
@@ -6,19 +6,23 @@ export type Variables = { +[name: string]: $FlowFixMe, ... };
 
 export type DataID = string;
 
-export interface RecordProxy {
-  copyFieldsFrom(source: RecordProxy): void;
-  getDataID(): DataID;
-  getLinkedRecord(name: string, args?: ?Variables): ?RecordProxy;
```

Comment:
@tbergq Why cannot we use the `Variables` type here? It's already in Runtime package.

Reply:
Probably not, this is just the result of a hasty copy-paste. I will fix it.

Reply:
🚀